tests/units/fastsync/assertions.py

```diff
@@ -100,8 +100,8 @@ def assert_sync_table_exception_on_failed_copy(sync_table: callable,
 
                     assert sync_table('table_1', FASTSYNC_NS) == 'table_1: Boooom'
 
-                    utils_mock.get_target_schema.assert_called_once()
-                    tap_mock.return_value.copy_table.assert_called_once()
+                    assert utils_mock.get_target_schema.call_count == 1
```

Comment:
why not `self.assertEqual`? Or that's more like another refactoring PR?

Reply:
Because there is no `self`, `self.assertEqual` can only be used when you inherit from `unittest.TestCase`. 
This is the [assertions](https://github.com/transferwise/pipelinewise/blob/master/tests/end_to_end/helpers/assertions.py) file which is just a bunch of functions.